### PR TITLE
Ignore empty SD card readers

### PR DESCRIPTION
**Pull request by johnterickson**
Some SD Card readers show a drive letter even when empty.  Instead of erroring out, let's just ignore those.

### CODE REVIEW

File: lib/Windows/VolumeInfoCache.cs [hunk 1]
```diff
@@ -94,6 +94,11 @@ public VolumeInfo GetVolumeForPath(string path)
         {
             if (path.IsSubpathOf(spv.SubPath))
             {
+                if (spv.Volume == null)
+                {
+                    break;
+                }
+
```
Comment (erikmav) [inline lib/Windows/VolumeInfoCache.cs]: Why break here? Seems like we just want to skip these at the foreach: `foreach (SubPathandVolume spv in subPathsAndVolumes.Where(s => s.Volume != null))`

File: lib/Windows/VolumeInfoCache.cs [hunk 2]
```diff
@@ -116,6 +124,11 @@ private static VolumeInfo GetVolumeInfo(VolumePaths volumePaths)
         if (!result)
         {
             int lastErr = Marshal.GetLastWin32Error();
+            if (lastErr == ERROR_NOT_READY || lastErr == ERROR_INVALID_PARAMETER)
+            {
+                return null;
```
Comment (erikmav) [inline lib/Windows/VolumeInfoCache.cs:128]: nit; Comment why this happens for future reference

**Review by erikmav — CHANGES_REQUESTED** [review]

**Review by erikmav — APPROVED** [review]

**Comment by johnterickson** [comment]
Tests passed locally with SD reader attached.

**Comment by erikmav** [comment]
Package publish of 0.3.1 proceeding, see #22

**Pull request merged**